swift/Sources/Addresses/EthereumAddressExtension.swift

```diff
@@ -8,6 +8,10 @@ import Foundation
 
 /// Ethereum address.
 extension EthereumAddress: Address, Hashable {
+    public convenience init?(string: String) {
+        self.init(string: string, type: .eip55)
```

Comment:
how to create wanchain address checksum from string?

Reply:
`EthereumAddress(string: string, type: .wanchain)`